tests/yast2_gui/yast2_system_settings.pm

```diff
@@ -0,0 +1,89 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: Run yast2 system_settings module, attempt to add a non existing
+# driver in the PCI ID setup and validate the error message. Disable the sysrq
+# keys at the kernel settings and validate the change in the system configuration.
+# Enable sysrq and validate again.
+#
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "y2_module_guitest";
+use strict;
+use warnings;
+use testapi;
+use YuiRestClient;
+use YaST::Module;
+use y2lan_restart_common 'close_xterm';
+use utils 'zypper_call';
+use x11utils 'start_root_shell_in_xterm';
+
+my $system_settings;
+
+sub pre_run_hook {
+    my ($self) = @_;
+    $system_settings = $testapi::distri->get_system_settings();
+    $self->SUPER::pre_run_hook;
```

Comment:
I don't think this line is needed, it is correct though, only thing is the order, always at the end invoke the parent method.

Reply:
yes, the correct way is to call the parent but as far as I could check we are not doing anything that we need so we can overwrite the method completely removing both lines.